src/middlewared/middlewared/client/client.py

```diff
@@ -463,7 +478,7 @@ def from_json(args):
                     if e.error:
                         print(e.error, file=sys.stderr)
                     if e.trace:
-                        print(e.trace['formatted'], file=sys.stderr)
+                        print(e.trace, file=sys.stderr)
```

Comment:
As I was developing it, I had tracebacks and they would then get bottlenecked here.

It would give the following error: "str has no property formatted error".

I'll try to see if I can get you an example.